src/content-helper/common/css/variables.scss

```diff
@@ -7,6 +7,7 @@ $html-font-size: 16px; // Used in functions.scss.
  * variables in the end of the file.
  */
 .settings_page_parsely,
+.components-panel,
```

Comment:
It's needed because we need to have the variables available in the block inspector panel, which is outside the PCH sidebar. I don't believe it's affecting any of the core panels, but I will look into it and see if I can get more specific in the selector, to avoid any potential issues.